tests/testthat/test-shinytest2-tm_front_page.R

```diff
@@ -0,0 +1,65 @@
+test_that("e2e: tm_front_page initializes without errors", {
+  skip_if_too_deep(5)
+  require(shinytest2)
+
+  data <- simple_cdisc_data()
+  data <- within(data, {
+    attr(ADSL, "metadata") <- list("Author" = "NEST team", "data_source" = "synthetic data")
+  })
+
+  app <- TealAppDriver$new(
+    data = data,
+    modules = teal::modules(
+      tm_front_page(
+        header_text = c(
+          "Important information" = "It can go here.",
+          "Other information" = "Can go here."
+        ),
+        tables = list("MTCARS" = head(mtcars, 5), "IRIS" = head(iris, 5)),
+        additional_tags = HTML("Additional HTML or shiny tags go here <br>"),
+        footnotes = c("X" = "is the first footnote", "Y is the second footnote"),
+        show_metadata = TRUE
+      )
+    ),
+    timeout = 3000
+  )
+
+  app$wait_for_idle()
+  app$expect_no_shiny_error()
+
+  # header text
+  testthat::expect_identical(
+    rlang::hash(app$get_html("#front_page_headers")),
+    "2b2ab743c68213927d0578b7e39f5873"
+  )
```

Comment:
Instead of a hash, how about using `app$expect_html(#front_page_headers)`?

It would be easier to update

The same applies to the other functions with a mix of existing `app$expect_*` and `testthat::expect_snapshot()` for the `app$get_active_module_output`

Unless you want to create this method first upstream at `{teal}`

Reply:
done [15e409c](https://github.com/insightsengineering/teal.modules.general/pull/717/commits/15e409cbd7648dcef1903d5b859f552a14c6dbc7)

Reply:
oh my, this hash idea seems an overkill : P